include/seqan3/io/detail/misc_input.hpp

```diff
@@ -105,7 +105,7 @@ inline auto make_secondary_istream(std::basic_istream<char_t> & primary_stream,
     if (read_chars == magic_number.size() && contrib::_bgzfCheckHeader(magic_number.data())) // BGZF
     {
     #ifdef SEQAN3_HAS_ZLIB
```

Comment:
Can you please indent the corresponding `#else` and `#endif` statements below to the same depth as this line?

Reply:
Done